### PR TITLE
US81437: remove d2l-alert and make it a bower component

### PR DESCRIPTION
**Pull request by cgalvind2l**
Acceptance Criteria:

    d2l-alert lives independently from d2l-my-courses-ui as a Polymer web component
    d2l-my-courses-ui is refactored to use d2l-alert as a dependency
    nothing breaks

d2l-alert now lives here: https://github.com/Brightspace/alert

Intention is to someday move it to BrightspaceUI

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "iron-scroll-threshold": "PolymerElements/iron-scroll-threshold#^1.0.2",
     "iron-pages": "^1.0.8",
     "neon-animation": "^1.2.4",
-    "polymer": "^1.7.0"
+    "polymer": "^1.7.0",
+    "d2l-alert": "git://github.com/Brightspace/alert.git#master"
```
Comment (CodeBaboon) [inline bower.json]: nit, but it would be nice if this were up on line 32

**Comment by CodeBaboon** [comment]
Outside of this repo, but d2l-alert repo could use a readme and a demo

**Comment by cgalvind2l** [comment]
The README and demo is next on my list :)

File: bower.json (hunk 1 above)
Comment (CodeBaboon) [inline bower.json]: Oh and it would behoove us to do a release of d2l-alert and then link to that instead of master before we put this to BSI, I suspect.

Comment (cgalvind2l) [inline bower.json]: OK; I'll add the README/Demo & do a release.

**Comment by CodeBaboon** [comment]
Tested this by pulling this branch, clearing `bower_components`, `bower install`, then doing the same for brightspace-integration. My courses still showed the unpinned alert correctly (no noticeable change in behavior). Seems legit.

**Comment by omsmith** [comment]
Don't want to actually register it as _d2l-alert_ just yet?

**Comment by cgalvind2l** [comment]
Honestly I assumed that putting something in BrightspaceUI would require "more" (work? documentation? signed-off-edness? something) than leaving it in Brightspace, but clearly that assumption was incorrect and I'll be putting it into BrightspaceUI etc today.

**Pull request merged**